src/SSL/Entities/ValidationResult.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace UKFast\SDK\SSL\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property bool $result
+ * @property array $altHosts
+ * @property int $expiryTimestamp
+ */
+class ValidationResult extends Entity
+{
+    //
```

Comment:
Please could we cast the expiry to a DateTime object using the `dates` property?